src/main/resources/skills.graphqls

```diff
@@ -15,14 +15,14 @@ type Query {
   grades: [Grade!]!
 }
 
-type Skill {
+type SkillDTO {
   id: ID!
   title: String!
   description: String!
-  examples: String!
-  categoryId: String!
+  examples: String
+  category: Category
   gradeId: String!
-  craftIds: [String!]!
```

Comment:
How come this isn't mandatory anymore - is there a scenario where we'd return a null for this field instead of an empty list?

Reply:
I might have not had my db set up correctly, but I was finding that it was throwing an error when fetching some skills which had no craftIds because we had set this to not be null. 
I think if the db has null craftIds, then graphql on its own can't convert that to an empty list, and so we need to convert that ourselves in the API.
So just to stop the erroring I modified it to be not null. I'd prefer to get this ticket in and raise another one to sort out the null -> list conversion. But I realise thats a bit cheeeky as I'm changing the desired behaviour so what do you think?

Reply:
Happy to sort it out on another ticket :)

Reply:
Great thanks :) 
https://devservices.jira.com/browse/CAA-56